src/mpi/datatype/typerep/src/typerep_yaksa_pack.c

```diff
@@ -323,3 +500,237 @@ int MPIR_Typerep_wait(MPIR_Typerep_req typerep_req)
   fn_fail:
     goto fn_exit;
 }
+
+int MPIR_Typerep_op(void *source_buf, int source_count, MPI_Datatype source_dtp,
+                    void *target_buf, int target_count, MPI_Datatype target_dtp,
+                    MPI_Op acc_op, int src_kind)
```

Comment:
Just `op` rather than `acc_op`. There is no `acc` context here.

`src_kind` is out-of-context here. Name it a boolean instead, e.g. `bool src_is_packed`. Then in the caller, we can pass it in as `src_kind == MPIDIG_ACC_SRCBUF_PACKED`.

It is probably helpful to add a comment here that only built-in op is supported.

Reply:
If I do this change, I need to change the caller as well, which I think is out of scope for this PR.